jwst/srctype/tests/test_srctype.py

```diff
@@ -175,9 +175,9 @@ def test_nrs_fixedslit():
 
     result = srctype.set_source_type(input)
 
-    assert(result.slits[0].source_type == 'POINT')
+    assert(result.slits[0].source_type == 'EXTENDED')
     assert(result.slits[1].source_type == 'EXTENDED')
-    assert(result.slits[2].source_type == 'POINT')
+    assert(result.slits[2].source_type == 'EXTENDED')
```

Comment:
Not your mistake, but `assert` is a statement, not a function, and does not need parenthesis. So:

```python
assert result.slits[0].source_type == 'EXTENDED'
assert result.slits[0].source_type == 'EXTENDED'
assert result.slits[0].source_type == 'EXTENDED'
```
is preferable

Reply:
I like the idea of changing the unit test so that the primary slit is now POINT, just to make it different from the others.